e2e/index.js

```diff
@@ -0,0 +1,20 @@
+require('dotenv').config({ path: process.env.DOTENV_CONFIG_PATH });
+
+console.log('');
+console.log('Console.log in index.js');
+
+const u = process.env.USER_NAME.split('@');
```

Comment:
hmm. why split at the `@`?

Reply:
agreed. I think its fine the print the entire email, reduces the chance of a bug if all instances of the user name use the fully qualified name.

Also, the domain name can be useful for debugging in some scenarios.

Reply:
If the email is not split, CircleCI thinks emails are confidential secrets so text are masked with `************`. To workaround this issue, I split the email and log array to Console.